securesystemslib/util.py

```diff
@@ -246,7 +253,8 @@ def ensure_parent_dir(filename, storage_backend=None):
     storage_backend.create_folder(directory)
 
 
-def file_in_confined_directories(filepath, confined_directories):
+def file_in_confined_directories(filepath: str,
+    confined_directories: Union[List[str], Tuple[str]]) -> bool:
```

Comment:
The tuple case seems weird and completely unused ... but I think your annotation does not work as it defines a tuple of specifically one item.

I think this covers both cases: `confined_directories: Sequence[str]` -- docs say _Tuples implement all of the common sequence operations._ and that should be good enough for us since we only need `x in y`

Could update the method documentation as well.

Reply:
Seems like the correct annotation was `Tuple[str, ...]` but I agree that `Sequence` works just as well and makes things look cleaner.